asreview/webapp/src/ProjectComponents/ReviewComponents/ReviewPage.js

```diff
@@ -172,19 +173,19 @@ const ReviewPage = () => {
       )}
 
       {isError && (
-        <Box sx={{ textAlign: "center", mt: 4 }}>
-          <Typography variant="h6" color="error" gutterBottom>
+        <Alert severity="warning" sx={{ mt: 3 }}>
```

Comment:
```suggestion
        <Alert severity="error" sx={{ mt: 3 }}>
```

This catches an error, so the severity needs to be error